man/ocamlrun.m

```diff
@@ -69,22 +69,45 @@
 .B b
 flag in the OCAMLRUNPARAM environment variable (see below).
 .TP
+.B \-H
+Allocates heap chunks by mmapping huge pages. Huge pages are locked into
+memory, and are not swapped.
+.TP
 .BI \-I \ dir
 Search the directory
 .I dir
 for dynamically-loaded libraries, in addition to the standard search path.
 .TP
+.B \-m
+Print the magic number of the bytecode executable given as argument and
+exit.
+.TP
+.B \-M
+Print the magic number expected for bytecode executable by this version
+of the runtime and exit.
+.TP
 .B \-p
 Print the names of the primitives known to this version of
 .BR ocamlrun (1)
 and exit.
 .TP
+.B \-t
+Increments debug trace level (ignored unless debug mode)
+.TP
 .B \-v
 Direct the memory manager to print verbose messages on standard error.
 This is equivalent to setting
 .B v=63
 in the OCAMLRUNPARAM environment variable (see below).
 .TP
+.B \-w
+Sets size of window used by major GC for smoothing out variations in
+its workload. This is an integer between 1 and 50. (Default: 1)
+.TP
+.B \-W
+Activates runtime warnings (such as Channel opened on file dies without
+being closed, unflushed data, etc.)
+.TP
```

Comment:
Again, these are `OCAMLRUNPARAM` settings, not `ocamlrun` command line arguments.